src/Bicep.LangServer/SemanticTokenVisitor.cs

```diff
@@ -173,6 +172,12 @@ public override void VisitModuleDeclarationSyntax(ModuleDeclarationSyntax syntax
             base.VisitModuleDeclarationSyntax(syntax);
         }
 
+        public override void VisitIfExpressionSyntax(IfExpressionSyntax syntax)

```

Comment:
There's another `SemanticTokenVisitor` in the Bicep.Wasm project that you'll need to update

Reply:
Thanks that's good to know. Updated it as well.